FirebaseFunctions/Sources/Functions.swift

```diff
@@ -104,6 +104,10 @@ internal enum FunctionsConstants {
     return HTTPSCallable(functions: self, name: name)
   }
 
+  @objc(HTTPSCallableWithUrl:) open func httpsCallable(url: String) -> HTTPSCallable {
```

Comment:
It could be confusing to have a String parameter named url. Any reason for the API not taking a URL parameter (NSURL in Objective C)?